Magical-Kitties/Magical-Kitties-Save-the-Day.css

```diff
@@ -137,23 +151,16 @@ input[type="text"] {
 }
 
 textarea {
-  width: 100%;
   line-height: 32px;
-  
 }
 
 input[type="text"], select, textarea {
   vertical-align: top;
 }
 
-
-
-/* Adjusting repeating-item button placement */
-
-
 /* Font & text color-related formatting */
 
-h1, h2, h3, h4, h5, h6, .redtext, .pinktext, .whiteonredtext {
```

Comment:
h1-6 styling was overriding elements in other sheet tabs, so removed them from here and relied on class names instead